{"name": "test_login[data1]", "status": "failed", "statusDetails": {"message": "AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nassert None is not None", "trace": "@pytest.fixture(scope=\"session\", autouse=True)\n    def init_token():\n        \"\"\"初始化环境配置\"\"\"\n        cfg = read_yaml(\"config/config.yaml\")\n>       assert cfg is not None, \"配置文件 config/config.yaml 读取失败，请检查路径或内容\"\nE       AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nE       assert None is not None\n\nconftest.py:9: AssertionError"}, "description": "\n    用户登录测试\n    ", "parameters": [{"name": "data", "value": "{'name': 'test002', 'password': '123456'}"}], "start": 1761027515554, "stop": 1761027515554, "uuid": "7acb430e-9843-4be2-81f6-21a8420ec132", "historyId": "ef4e97727be586459ef7c24e8360cd74", "testCaseId": "ab4ae781b39039ec4eae113a075c2b68", "fullName": "testcases.test_user_login#test_login", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_user_login"}, {"name": "host", "value": "DESKTOP-UGQEI6T"}, {"name": "thread", "value": "11844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_user_login"}], "titlePath": ["testcases", "test_user_login.py"]}